jobserver/models/project.py

```diff
@@ -197,3 +198,14 @@ def title(self):
             return self.name
 
         return f"{self.number} - {self.name}"
+
+    @property
```

Comment:
Just to be a pain (I should have thought about this on Friday), but I think you can improve the performance in some cases by using the Django `cached_property` [decorator](https://docs.djangoproject.com/en/5.0/ref/utils/#module-django.utils.functional) instead. It doesn't seem to make a difference in many cases, but for the [event log](http://localhost:8000/event-log/) page for an unauthenticated user it takes the number of queries down from 105 to 14 (approximately).